welkin/models/pagination.py

```diff
@@ -0,0 +1,87 @@
+from welkin.models.base import PageIterator
+
+
+class PageableIterator(PageIterator):
+    """
+    Most common paging class
+    """
+
+    def __init__(self, collection, resource, method, size=20, *args, **kwargs):
```

Comment:
No need to have all of the arguments listed in these subclasses. You can just do:

```python
def __init__(self, *args, **kwargs):
    super().__init__(*args, **kwargs)
    ...
```